src/components/guide-tooltip/GuideTooltip.tsx

```diff
@@ -2,15 +2,15 @@ import * as React from 'react';
 import { FormattedMessage } from 'react-intl';
 import classNames from 'classnames';
 
-import Tooltip, { TooltipTheme } from '../tooltip';
+// @ts-ignore flow import
```

Comment:
Do we need this?

Reply:
yes it needed for the cause `TooltipProps` is not passing ts linting
it has to be ignored in flow to pass all linting errors

Reply:
This file is in TS, and the import is in TS, so it shouldn't need an ignore